src/loader-components/PropsTable.module.scss

```diff
@@ -14,4 +15,13 @@
     white-space: normal;
     word-wrap: break-word;
   }
+
+  tr {
+    background-color: var(--terra-dev-site-props-table-background-color, #fff);
```

Comment:
scope these to the `tr` to be safe and prevent collisions in the future. 

`--terra-dev-site-props-table-tr-background-color`